src/main/java/de/torui/coflsky/utils/ChatUtils.java

```diff
@@ -1,10 +1,25 @@
 package de.torui.coflsky.utils;
 
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.ChatStyle;
+import net.minecraft.util.EnumChatFormatting;
+import net.minecraft.util.IChatComponent;
+
 public class ChatUtils {
 
     public static String cleanColour(String in) {
         return in.replaceAll("(?i)\\u00A7.", "");
     }
+    public static IChatComponent getCoflnetLogo(boolean includeBrackets){
```

Comment:
please create two different methods where the method name says if it has brackets or not